firefox-ios/firefox-ios-tests/Tests/XCUITests/FakespotTests.swift

```diff
@@ -66,6 +66,33 @@ class FakespotTests: IphoneOnlyTestCase {
         mozWaitForElementToNotExist(app.buttons[AccessibilityIdentifiers.Toolbar.shoppingButton])
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2358924
+    // Smoketest
+    func testAcceptTheRejectedOptInNotification() {
```

Comment:
This test has failed 1 out of 3 in the PR, can we try to make it more stable? 
~Tests/XCUITests/BaseTestCase.swift:128 - failed - Timed out waiting for element "Search Amazon" TextField to exist

Reply:
yea, i am planning to look into it, locally is hard to catch that corner case, but i will make it more stable